utilities/rali/rali_dataloader_mt/README.md

```diff
@@ -0,0 +1,25 @@
+# rali_dataloader_mt application
+This application demonstrates a basic usage of RALI's C API to use sharded data_loader  in a multithreaded application.
+<p align="center"><img width="90%" src="../../docs/images/image_augmentation.png" /></p>
+
+## Build Instructions
+
+### Pre-requisites
+* Ubuntu Linux, [version `16.04` or later](https://www.microsoft.com/software-download/windows10)
+* RALI library (Part of the MIVisionX toolkit)
+* [OpenCV 3.4+](https://github.com/opencv/opencv/releases/tag/3.4.0)
+* Radeon Performance Primitives (RPP)
```

Comment:
Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5745979)

Reply:
Pushed the changes to README. I found that all other README in rali utilities will have the same issue